plugins/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -36,7 +36,7 @@
     default: 'A'
     choices: ['A', 'AAAA', 'A6', 'CNAME', 'DNAME', 'MX', 'PTR', 'SRV', 'TXT']
     type: str
-  record_value:
+  record_values:
```

Comment:
This need to stay `record_value`, and you have to add new documentation for `record_values`.